docs/source/reference/api/aieng_template.bar.rst

```diff
@@ -0,0 +1,7 @@
+aieng\_template.bar module
```

Comment:
the files under reference/api can be removed. When you run the `make html` command after specifying the top level rst files, it will be automatically generated